test/runtests.jl

```diff
@@ -1,5 +1,6 @@
 module PlotsTests
 
+include("add_packages.jl")
```

Comment:
I feel bad about unconditionally installing packages via runtests.jl which could ruin users' environments.  Something like

```julia
if isinteractive()
    @info """
    Skip installing unregistered packages.  To run Plots.jl test suite
    in interactive session, you need to install them manually by, e.g.,
    `include("test/add_packages.jl")`.  Alternatively, you can run the test
    with `Pkg.test("Plots")`.
    """
else
    include("add_packages.jl")
end
```

would be nice.  However, this still is not enough when the test is invoked from shell like `julia test/runtests.jl`.  Checking `Base.HOME_PROJECT` and `Base.ACTIVE_PROJECT` may solve this problem but I guess we need to be careful not to exclude their values when invoked via `Pkg.test`.